include/drivers/pwm.h

```diff
@@ -59,11 +59,15 @@ __subsystem struct pwm_driver_api {
 
 /**
  * @brief Set the period and pulse width for a single PWM output.
+ * 
+ * Passing zero to the pulse disables the PWM signal.
+ * The pin is driven to the inactive state for a disabled signal.
```

Comment:
API meeting:

Passing 0 to the @param pulse will cause the pin to be driven to a constant inactive level.
Passing a @param pulse equal @param period will  cause the pin to be driven to a constant active level.